pkg/broker/broker.go

```diff
@@ -261,16 +304,57 @@ func (a AnsibleBroker) Bootstrap() (*BootstrapResponse, error) {
 				log.Errorf("We have a plan that did not get its id generated: %v", p.Name)
 				continue
 			}
-			planNameManifest[p.ID] = p.Name
 		}
 	}
-	if err := a.dao.BatchSetSpecs(specManifest); err != nil {
-		return nil, err
+	return specManifest
+}
+
+// getSafeToDeleteSpecs - will check if any bundle instance spe
+func getSafeToDeleteSpecs(a AnsibleBroker, markedSpecs map[string]*bundle.Spec) []*bundle.Spec {
+	safeToDeleteSpecs := make([]*bundle.Spec, 0)
+	bundleInstances, err := a.dao.BatchGetBundleInstances()
+	if err != nil {
+		log.Errorf("error getting bundle instances '%+v'", err)
+		// returning nil because the instead of checking the error,
```

Comment:
rephrase: 'returning nil because the instead' should read 'returning nil instead of ...'

Reply:
done